certificates/views.py

```diff
@@ -74,6 +75,15 @@ def get_context_data(self, **kwargs):
         context['name'] = certificate.user.profile.full_name
         context['signatories'] = list(signatories)
         context['certificate'] = certificate
+        context[
+            'linkedin_share_context'
+        ] = 'certId={certId}&issueMonth={issueMonth}&issueYear={issueYear}&name={certName}&organizationId={organizationId}&isFromA2p=true'.format(
+            certId=certificate.hash,
+            issueMonth=certificate.created_on.month,
+            issueYear=certificate.created_on.year,
+            certName=course.title,
+            organizationId="2746406",
+        )
```

Comment:
I think we can move this chunk to a common method since all the values and string format is the same for both `Courses` and `Programs` except `certName` which we can pass to that LinkedIn context generator method.

Up to you if you decide to create the method in the parent class https://github.com/mitodl/micromasters/blob/hamza/4805-add_certificate_sharing_context/certificates/views.py#L27 to dynamically use it or as an independent function.

Reply:
Good suggestion, but I doubt that we will ever be changing the organizationID so it's not worth making it configurable.